src/main/java/com/disraely/entities/Employee.java

```diff
@@ -0,0 +1,28 @@
+package com.disraely.entities;
+
+import com.disraely.enums.EmployeeLevel;
+
+public abstract class Employee extends BaseEntity {
+    private Double salary;
+    private EmployeeLevel employeeLevel;
```

Comment:
I think it is easier to handle numbers for the experience rather than using enums.

For example:
from 1 to 10 or 1 to 100 or whatever you decide, could be the level of experience of this employee.

Reply:
Resolved!